src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -1,24 +1,58 @@
-﻿@using SFA.DAS.EmployerAccounts.Web.Extensions
+@using SFA.DAS.EAS.Portal.Client.Types;
 @model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
-
 @{
-    var vacancy = Model.AccountViewModel.Vacancies.First();
-}
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+    string status = "";
+
+    if (Model.HasSingleClosedVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Closed);
+        actionLinkText = "Manage vacancy";
+        numberOfApplications = vacancy.NumberOfApplications;
+        status = "Closed";
+    };
 
+    if (Model.HasSinglePendingReviewVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Submitted);
+        actionLinkText = "Preview vacancy";
+        status = "Pending review";
+    };
+
+    if (Model.HasSingleDraftVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.Draft);
+        actionLinkText = "Edit and submit vacancy";
+        status = "Draft";
+    };
+
+    if (Model.HasSingleRejectedVacancy)
```

Comment:
The way the `HasSingleXxxVacancy` properties on the model are calculated, it looks like you could have e.g. 4 vacancies, 1 each of the 4 statuses checked, so all the properties are set to true, then here, the last status checked would be true (`HasSingleRejectedVacancy`), so the 4 returned vacancies would be interpreted as a single rejected vacancy.

The changes to this view also breaks the existing handling of the draft status vacancy.

This panel is only shown when there is a single vacancy, so all it has to do is check `vacancy.Status` (and the `HasSingleXxxVacancy` properties can be removed.

Reply:
The panel is only called from the controller if there is a single vacancy.  The rule in the view are just UI rules.

Reply:
Yes, I realised that after writing the comment, so what's here should work.

We could tidy it up by taking the first vacancy, and using it's status, but that's a tidy up, as opposed to an issue.

Reply:
I've refactored to tidy up.